config/passport.js

```diff
@@ -1,26 +1,3 @@
 const passport = require("passport");
 const LocalStrategy = require("passport-local").Strategy;
-const mongoose = require("mongoose");
-const User = mongoose.model("User");
 
-passport.use(
```

Comment:
please revert this change.
I don't think all this code should be removed.

Reply:
Also, fix all merge conflicts by rebasing to develop.

Reply:
The changes have been reverted

Reply:
The changes have been reverted and I've rebased